app/assets/javascripts/base.js.erb

```diff
@@ -4,6 +4,7 @@
 //= require initializePage
 //= require utilities/getImageForLink
 //= require @honeybadger-io/js/dist/browser/honeybadger.js
+//= require @hotwired/turbo
```

Comment:
This is causing the specs to fail. If I remove this locally and run a spec that's failing on Travis it will pass. If I put it back, it fails. So confused. 

Spec I'm running: `bundle exec rspec spec/system/user/user_self_destroy_spec.rb`

failure:
```
 1) User destroys their profile destroys an account
     Failure/Error: expect(matching_job).to be_present, "No enqueued job found with #{expected}"
       No enqueued job found with {:job=>Users::DeleteWorker}
```

Reply:
That particular rspec test passes for me locally on this branch, but I did notice a lot of the failures (especially in Cypress) relate to user creation/ sign in / sign out. 

Running the app locally I also see an issue when I sign in - I'm initially not presented with any feed and have to refresh the page (this happens when signing up a new user too):

https://user-images.githubusercontent.com/20773163/159679388-20b34446-edf0-4166-98ab-b45b38a4061f.mp4

Similar to what you found with the specs, if I remove this line, everything works as normal. I haven't had a chance to dig into it any further, but thought I'd mention it as sometimes having something reproducible in the browser can help with debugging!

Reply:
In this area, I'm out of my element.  What I'm wondering if we need turbo declared earlier?  For example, I'm looking at `//= require initializePage` on line 4 and wondering if there's a temporal dependency/bug.

Just thinking in a general sense the turbo require is more conceptually global than initializePage.

Reply:
should the job value be a class name string and not a class constant? (sidekiq's api relies on json serialization, not sure if that's coming into play, and no clue at all why that would happen only in travis)

Reply:
(My draft PR moved the declaration and that didn't help)

Reply:
I locally tested the following:

```ruby
    sidekiq_assert_enqueued_with(job: "Users::DeleteWorker") do
      click_button "Delete account"
    end
```

I get the following:
```
  1) User destroys their profile destroys an account
     Failure/Error:
       matching_job = job.jobs.detect do |queued_job|
         expected_args.all? { |key, value| value == queued_job[key] }
       end

     NoMethodError:
       undefined method `jobs' for "Users::DeleteWorker":String
```

Reply:
<img width="1025" alt="Screen Shot 2022-03-18 at 3 26 47 PM" src="https://user-images.githubusercontent.com/2130/159722558-372a8c55-5b8f-4481-9782-5916f187ca38.png">

To triangulate, it is as though the flash message isn't being rendered for the above

Reply:
@fdocr I believe you're right! I've added `Turbo.session.drive = false` and that cleared up the issues for me locally. Including the sign in weirdness. 😄 

the thing I'm kind of on the fence about is that I didn't add the `data` element to the turbo frame and it still seemingly works; I would think it would fail. But, we are just using a `<turbo-frame />` instead of an anchor tag or form. I'll test this a bit more before giving it the 👍 

Any way, huge thanks to you all! 🙏 @aitchiss @jeremyf @djuber